scripts/main/settings.js

```diff
@@ -483,3 +483,47 @@ settings.save_enter = function (e) {
 		},
 	});
 };
+
+settings.viewToken = function () {
+	api.post("User::getCurrent", {}, function (data) {
+		let bodyHtml = "";
+		let enableReset = "";
+		if (data.token === "") {
+			bodyHtml = `<div class='directLinks'><p>${lychee.locale["DISABLED"]}</p></div>`;
+			enableReset = lychee.locale["ENABLE"];
```

Comment:
What is `enableReset`?

Reply:
Then it should be named like that, i.e. `buttonCaption` or `submitButtonCaption`, or, or, ...

Reply:
Ah, I now see I have already done that. :+1: